domain/model/bootstrap/bootstrap.go

```diff
@@ -70,12 +89,25 @@ func CreateModel(
 				return fmt.Errorf("determining cloud type for model %q: %w", args.Name, err)
 			}
 
-			if err := state.Create(ctx, tx, modelType, args); err != nil {
-				return fmt.Errorf("create bootstrap model %q with uuid %q: %w", args.Name, args.UUID, err)
+			if args.SecretBackend == "" && modelType == coremodel.CAAS {
```

Comment:
This should not be allowed to be empty. A secret backend is always needed to be specified. The default is "auto".

Reply:
This is exactly how it was implemented in the state layer. If this is wrong then I am going to push it back on yourself for the correct implementation.